docs/src/docs-table.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { ReactElement } from "react";
+import styled, { css } from "styled-components";
+import {
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  mediaQueries as mq,
+} from "@kiwicom/orbit-components";
+import { StyledTable } from "@kiwicom/orbit-components/lib/Table";
+import { StyledTableCell } from "@kiwicom/orbit-components/lib/Table/TableCell";
+import { StyledTableRow } from "@kiwicom/orbit-components/lib/Table/TableRow";
+import { StyledTableBody } from "@kiwicom/orbit-components/lib/Table/TableBody";
+
+export const TableHeadCell = styled(StyledTableCell)`
+  background-color: ${({ theme }) => theme.orbit.paletteCloudLight};
+  text-align: left;
+`;
+
+const Table = styled(StyledTable)`
+  border-collapse: separate;
+  border-spacing: 0;
+
+  ${StyledTableCell} {
+    border-width: 1px;
+    border-color: ${({ theme }) => theme.orbit.borderColorTable};
+    border-style: none none solid none;
+  }
+
+  ${TableHeadCell} {
+    border-style: solid none none none;
+
+    &:first-child {
+      border-top-left-radius: 6px;
+      border-left-style: solid;
+    }
+
+    &:last-child {
+      border-top-right-radius: 6px;
+      border-right-style: solid;
+    }
+  }
+  ${StyledTableBody} > ${StyledTableRow} {
+    transition: transform ease-in-out 0.2s;
+
+    &:hover {
+      background-color: inherit;
+
+      ${mq.desktop(css`
+        border-radius: 6px;
+        transform: scale(1.025);
+        box-shadow: 0px 4px 8px 0px #252a311f, 0px 1px 4px 0px #252a3129;
+
+        ${StyledTableCell} {
+          border-top-style: solid;
```

Comment:
I do not think you need that, it's causing a layout shift and the whole table jumps a bit 🤔 

maybe you can also consider using box-shadow instead of borders, so it will be more smooth for such cases when it will be hovering.

 like `box-shadow: 1px 0 0 0 token` and on hover just to use normal box-shadow elevation token